src/data/two_pointing.lean

```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import data.sum.basic
+import logic.nontrivial
+
+/-!
+# Two-pointings
+
+This file defines `two_pointing α`, the type of two pointings of `α`. A two-pointing is the data of
+two distinct terms.
+
+This is morally a Type-valued `nontrivial`. Another type which is quite close in essence is `sym2`.
```

Comment:
Ah, so it's an `unsym2` :smile:

(Is there another name for `two_pointing`? It's not bad, but I suspect pairs of distinct points must have been called other things before...)

Reply:
Probably! "Distinct ordered pair" would be one. `two_pointing` has the advantage of being very specific to this type, while other names tend to be a little vaguer.

Reply:
Another option: `dpair` for "distinct pair" (though it might be mistaken for "dependent pair")

Reply:
Yeah, I'm not convinced.